.github/CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+# Default code owners for fuel-core
+* @xgreenx @Dentosal @MitchTurner
```

Comment:
perhaps the older team members want more granularity on which parts they've written majorly ?

Reply:
I think this is fine as-is, no reason to complicate this for now. If I want someone else to do review for a specific part of code, I'll just ask them to review instead.

Reply:
Yeah I figured let's start simple and extend it when we want to be more specific. The long term vision imo is that we have the client team as default code owners for the repo, and more well-defined sub-teams for any specific part of the code we want to have more granular control over.